composer.json

```diff
@@ -24,6 +24,19 @@
                     "type": "zip"
                 }
             }
+        },
+        {
+            "type": "package",
+            "package": {
+              "name": "islandora/views_nested_details",
+              "version": "1.0.0-beta1",
+              "type": "drupal-module",
+              "source": {
+                "url": "https://github.com/Islandora-Labs/views_nested_details",
+                "type": "git",
+                "reference":"1.0.0-beta1"
```

Comment:
Is this section a workaround because it's not on packagist? Should we just put it on packagist?

Second, can we require a version range rather than a specific version? Relatedly, should we release a 1.0?